qiskit_experiments/library/characterization/t1.py

```diff
@@ -100,34 +101,44 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
             The experiment circuits
 
         Raises:
-            AttributeError: if unit is dt but dt parameter is missing in the backend configuration
+            AttributeError: if unit is `dt`, but `dt` parameter
+                is missing in the backend configuration.
         """
+        conversion_factor = 1
         if self.experiment_options.unit == "dt":
             try:
-                dt_factor = getattr(backend.configuration(), "dt")
+                dt_factor = getattr(backend._configuration, "dt")
+                conversion_factor = dt_factor
             except AttributeError as no_dt:
                 raise AttributeError("Dt parameter is missing in backend configuration") from no_dt
+        elif self.experiment_options.unit != "s":
+            conversion_factor = apply_prefix(1, self.experiment_options.unit)
+
+        self.set_analysis_options(
+            extra={
+                "conversion_factor": conversion_factor,
+                "unit": self.experiment_options.unit,
```

Comment:
These extra metadata are really necessary? For example, spectroscopy experiment doesn't provide such extra data (usually people don't care the conversion factor and specified unit if result is reasonable).

Reply:
It sounds like it's necessary in order to repeat the experiment. @wshanks what do you think?

Reply:
I don't remember what's stored where, also @chriseclectic plans to make some changes. If this information is available elsewhere then it surely can be removed from here.